zwave_js_server/const.py

```diff
@@ -128,3 +129,48 @@ class CommandClass(IntEnum):
     ZIP_ND = 88
     ZIP_PORTAL = 97
     ZWAVEPLUS_INFO = 94
+
+
+# Lock constants
+STATE_LOCKED = "locked"
```

Comment:
where is this one used ?

Reply:
I think it's a home assistant state, so we shouldn't add it here.

Reply:
I will remove, I was debating whether or not it belonged here